docs/reference/rest_api_index.md

```diff
@@ -28,6 +28,15 @@ curl -k https://127.0.0.1:5000/api/records?access_token=API-TOKEN
     The `-k` or `--insecure` option here is simply because a certificate is typically not setup locally. In production,
     your `curl` calls shouldn't need this option because you will be using a valid certificate.
 
+_Introduced in InvenioRDM v12_
```

Comment:
I think pointing out the content-type is good info, although I think it's been around for a while (v9 at least I think). In fact, investigating the practical difference between using `application/vnd.inveniordm.v1+json` and `application/json` and documenting that would be the nugget of value here. I *think* one of them does the ui serialization or does it differently or something like that... Let's document that aspect and we won't have to version mark it either this way :+1: